packages/web/src/questionnaire/trackStructureWorkforce.ts

```diff
@@ -1,159 +1,163 @@
   {
     help: "Goes to track_structure_building_property if : has infos about codeNaf AND user_help == 'unknown' (newbie)",
     rules: [
       {
-        from: 'usedTracks',
+        from: DataMappingFrom.UsedTracks,
         id: 'codeNaf',
         dataField: 'codeNaf',
         conditions: [
           {
             type: 'codeNaf',
-            operator: 'exists'
+            operator: ConditionOperators.exist
           }
         ]
       },
       {
-        from: 'usedTracks',
+        from: DataMappingFrom.UsedTracks,
         id: 'user_help',
         dataField: 'user_help',
         conditions: [
           {
             type: 'user_help',
-            operator: '==',
+            operator: ConditionOperators.is,
             value: 'unknown'
           }
         ]
       }
     ],
-    next: { default: 'track_structure_building_property' }
+    next: { default: TrackId.BuildingProperty }
   },
   {
     help: "Goes to track_goals if : have infos about sector AND user_help == 'precise' (pro)",
     rules: [
       {
-        from: 'usedTracks',
+        from: DataMappingFrom.UsedTracks,
         id: 'codeNaf',
         dataField: 'codeNaf',
         conditions: [
           {
             type: 'codeNaf',
-            operator: 'exists'
+            operator: ConditionOperators.exist
           }
         ]
       },
       {
-        from: 'usedTracks',
+        from: DataMappingFrom.UsedTracks,
         id: 'user_help',
         dataField: 'user_help',
         conditions: [
           {
             type: 'user_help',
-            operator: '==',
+            operator: ConditionOperators.is,
             value: 'precise'
           }
         ]
       }
     ],
-    next: { default: 'track_goals' }
+    next: { default: TrackId.Goals }
   }
 ]
 
-export const workforce = {
-  id: 'track_structure_workforce',
+export const workforce: Track = {
+  id: TrackId.StructureWorkforce,
   category: 'myEntreprise',
   title: { fr: 'Mes effectifs' },
   label: { fr: 'Combien êtes-vous dans votre entreprise ?' },
   interface: {
-    component: 'buttons'
+    component: TrackComponents.Buttons
   },
   behavior: {
     multipleChoices: false
   },
   options: [
     {
       disabled: true,
-      value: { 'entreprise . effectif': 0, structure_sizes: ['PME', 'TPE'] },
+      value: { [Entreprise.Workforce]: 0, structure_sizes: ['PME', 'TPE'] },
       title: { fr: 'TPE ou PME' },
       label: { fr: '🧍‍♂employé.s' },
-      hasInput: 'number',
+      hasInput: HasInputOptions.Number,
       defaultInput: 1,
       inputMax: 249,
       inputMin: 1,
-      inputField: 'entreprise . effectif',
+      inputField: Entreprise.Workforce,
       inputCleaning: [
         {
-          operator: '<',
+          operator: ConditionOperators.inferior,
           conditionValue: 20,
           valueField: 'structure_sizes',
           value: ['PME']
         },
         {
-          operator: '>',
+          operator: ConditionOperators.superior,
```

Comment:
Again, not linked to this PR, but there is an obvious unhandled case when `entreprise . effectif = 20`. 

Not sure we should keep this `inputCleaning` property as `structure_sizes` is used nowhere I am aware of.

Reply:
Indeed !

Reply:
Ca mériterait  certainement une issue

Reply:
issue: ok pour en faire une issue séparée. Il faut que je puisse aussi vérifier de mon côté qu'il n'y aura pas d'effets de bord : sur certaines pistes de questions ces `InputCleaning` peuvent servir non pas au filtrage des résultats, mais à savoir quelle piste de question doit ou peut s'afficher ensuite.

Reply:
done here: https://github.com/betagouv/transition-ecologique-entreprises-widget/issues/239